ui/src/components/forms/UserFormContainer.js

```diff
@@ -1,7 +1,8 @@
 import React, { useState, useEffect } from 'react';
 import { customAxios as axios } from '../helpers/customAxios';
 import Form from './Form';
-import { Container, Center, VStack, Text } from '@chakra-ui/react';
+import { Container } from '@chakra-ui/react';
+import { DateTime } from 'luxon';
 
 const UserFormContainer = (props) => {
   const [formData, setFormData] = useState(null);
```

Comment:
there is a split second where the form loads and then redirects to an error page. i think we need to set a `isLoading` variable here to only render the form once isLoading is set to true (at the end of your useEffect).